strawberryfields/api/__init__.py

```diff
@@ -23,5 +23,6 @@
 from .connection import Connection, RequestFailedError
 from .job import InvalidJobOperationError, Job, JobStatus
 from .result import Result
+from .post_processing import number_expectation_pnr, number_variance_pnr
 
-__all__ = ["Connection", "Job", "Result"]
+__all__ = ["Connection", "Job", "Result", "number_expectation_pnr", "number_variance_pnr"]
```

Comment:
It feels a bit weird having the post-processing module in the `api` package. These are functions that are relevant to sampled results in general, whether they were generated via the API, local engine, or apps layer.